### PR TITLE
Bhavani/vz 4021

### PR DESCRIPTION
**Pull request by bhavaniravichandran**
# Description

using go-kit metrics package to push metric to prometheus

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -18,13 +19,15 @@ require (
...
+	github.com/prometheus/client_golang v1.11.0
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.6.1
-	go.uber.org/zap v1.16.0
-	golang.org/x/lint v0.0.0-20201208152925-83fdc39ff7b5
-	golang.org/x/tools v0.0.0-20210106214847-113979e3529a
+	github.com/stretchr/testify v1.7.0
+	go.uber.org/tools v0.0.0-20190618225709-2cfd321de3ee // indirect
+	go.uber.org/zap v1.19.1
```
Comment (anders-swanson) [inline go.mod:27]: do we want to bump zap and these others

File: go.mod [hunk 2]
```diff
@@ -40,12 +43,13 @@ require (
 
 replace (
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
-	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.2.1
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.7.0
```
Comment (anders-swanson) [inline go.mod:46]: why is this replace in twice?

Comment (bhavaniravichandran) [reply go.mod:46]: I ran go mod tidy now and it deleted one of them. I will push the new go.mod

File: go.mod (hunk 1 above)
Comment (bhavaniravichandran) [reply go.mod:27]: looks like 1.19.1 is the latest for zap

Comment (anders-swanson) [reply go.mod:26]: But do we have license approval for Zap, please check with mark

**Pull request merged**